runtime/jcl/common/criusupport.c

```diff
@@ -275,6 +311,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 
 		/* We can only end up here if the CRIU restore was successful */
 
+		isAfterCheckpoint = TRUE;
```

Comment:
I think this deserves to be separated from the code below. I suggest moving it before the blank line above so it immediately follows the 'restore' comment above.